client/src/extension.ts

```diff
@@ -60,59 +60,36 @@ export function activate(context: vscode.ExtensionContext) {
 
           // brew install location mac
           // intel
-          "/usr/local/bin/nu",
+          "/usr/local/bin",
           // arm
-          "/opt/homebrew/bin/nu",
+          "/opt/homebrew/bin",
 
           // native package manager install location
-          "/usr/bin/nu",
+          // standard location should be in `PATH` env var
+          //"/usr/bin/nu",
         ];
 
-        let found_nushell_path = "";
-        const home = os.homedir();
-
-        for (const cur_val of pathsToCheck) {
-          // console.log("Inspecting location: " + cur_val);
-          let constructed_file = "";
-          if (cur_val.startsWith("~/scoop")) {
-            // console.log("Found scoop: " + cur_val);
-            const p = path.join(home, cur_val.slice(1));
-            // console.log("Expanded ~: " + p);
-            const file = glob.sync(p, "debug").toString();
-            // console.log("Glob for files: " + file);
-
-            if (file) {
-              // console.log("Found some file: " + file);
-              // if there are slashes, reverse them to back slashes
-              constructed_file = file.replace(/\//g, "\\");
-            }
-          } else if (cur_val.startsWith("~")) {
-            constructed_file = path.join(home, cur_val.slice(1));
-            // console.log("Found ~, constructing path: " + constructed_file);
-          } else {
-            constructed_file = cur_val;
-          }
-
-          if (fs.existsSync(constructed_file)) {
-            // console.log("File exists, returning: " + constructed_file);
-            found_nushell_path = constructed_file;
-            break;
-          } else {
-            // console.log("File not found: " + constructed_file);
-          }
-        }
+        const found_nushell_path = which.sync("nu", {
```

Comment:
does this work on windows too? `nu` vs `nu.exe`?

Reply:
yes, the default `pathExt` on windows includes `.exe`, `.com`, `.bat`, `.cmd`, `.vbs` and others. it's the same behavior as the `which` command (e.g. from msys2 or cygwin). it searches the directories in the same way the shell searches a command. 

side note: on windows, you need install third party packages to get `which` command, but there's a similar `where` command that is distributed with the operating system itself. the behavior is slight different, in that `where` will return multiple matches while `which` only returns the first match.

Reply:
Oh, I thought this was a package that implemented the `which` functionality without requiring an external command. That's kind of a non-starter. We don't want to require any additional commands.

Reply:
> there's a similar `where` command that is distributed with the operating system itself

I should point out, `nu` shell has a `where` built in which shadows the Windows default `where` command. you need to explictly use `where.exe` from nu. alternatively, you can run it through cmd.

Reply:
> Oh, I thought this was a package that implemented the `which` functionality without requiring an external command.

it is. I was trying to clarify it's behavior follows exact the `which` command. not that it's implemented using external command. sorry I wasn't clear.

Reply:
oh, ok. If it doesn't require any more external dependencies, then that's fine.